README.md

```diff
@@ -147,55 +122,55 @@ Each SDK will auto generate API reference documentation using the respective lan
 | Go         | tbd                    | tbd            |
 
 > [!IMPORTANT]
-> Producing API reference documentation is the responsibility of an _implementer_
+> Producing API reference documentation is the responsibility of an _implementer_.
 
 ### Example Feature Usage
 
-Each SDK will **publish** example usage for _each_ implemented feature. This can either be included as a part of API reference documentation _or_ published separately
+Each SDK will **publish** example usage for _each_ implemented feature. This can either be included as a part of API reference documentation _or_ published separately.
 
 ## Test Vectors
 
 Test vectors ensure interoporability of features across SDKs and language implementations by providing common test cases with an input and expected output pair. They include both success and failure cases that can be vectorized.
 
-This repo serves as the home for all web5 feature related vectors. They are available in the [test-vectors](./test-vectors/) directory
+This repo serves as the home for all web5 feature related vectors. They are available in the [test-vectors](./test-vectors/) directory.
 
-The `tbdex` repo houses tbdex feature related vectors. They are available in the [test-vectors](https://github.com/TBD54566975/tbdex/test-vectors) directory
+The `tbdex` repo houses tbdex feature related vectors. They are available in the [test-vectors](https://github.com/TBD54566975/tbdex/test-vectors) directory.
 
-The `sdk-report-runner` repo consumes the output tests for these test vectors in each repo and generates a report - [report-runner](https://github.com/TBD54566975/sdk-report-runner)
+The `sdk-report-runner` repo consumes the output tests for these test vectors in each repo and generates a report - [report-runner](https://github.com/TBD54566975/sdk-report-runner).
 
-### Adding/Updating Vectors
+### Adding & Updating Test Vectors
 
-New test vectors should follow the standard [vector structure](./test-vectors/). Vectors are automatically validated against the JSON schema via CI.
+New test vectors should follow the standard [vector structure](./test-vectors/). Vectors are automatically validated against their [JSON Schema](https://json-schema.org/) via CI.
 
-Create a PR in this repo for adding / updating web5 test vectors
+Create a PR in this repo for adding / updating Web5 test vectors.
 
 ### Feature Completeness By SDK
 
-Test vectors are also used to determine feature completeness via our [test harness](./test-harness/README.md). Results of test harness runs can be found [here](https://tbd54566975.github.io/sdk-development/).
-
 ## Web5 SDK Features
 
+Test vectors are used to determine feature completeness via our [test harness](./test-harness/README.md). Results of test harness runs can be found [here](https://tbd54566975.github.io/sdk-development/).
```

Comment:
Both links here are broken, probably fine but maybe we should track the work on an issue @frankhinek?

Reply:
I will update this to https://tbd54566975.github.io/sdk-report-runner/